kf_lib_data_ingest/validation/test_input_structure.py

```diff
@@ -0,0 +1,230 @@
+from dateutil.parser import parse as parsedate
```

Comment:
The name of this file doesn't seem to match what the content is doing. Can we rename this module to something else, maybe `value_validation.py` or could even do `values.py` since its within the `validation` package?

Reply:
Makes sense, no problem